src/lib/y2journal/entry_presenter.rb

```diff
@@ -25,9 +27,10 @@ module Y2Journal
   class EntryPresenter < SimpleDelegator
     include Yast::I18n
 
-    TIME_FORMAT = "%b %d %H:%M:%S".freeze
+    TIME_FORMAT = "%b %d %H:%M:%S"
 
     def initialize(entry)
+      super
```

Comment:
BTW this seems to duplicate `__setobj__` below, but [`SimpleDelegator` documentation](https://ruby-doc.org/3.4.1/stdlibs/delegate/SimpleDelegator.html) is awful, so let's leave this as it is

Reply:
Rubocop complained about it.